sdk/rust/oak/src/lib.rs

```diff
@@ -519,9 +519,17 @@ pub fn run_event_loop<T: crate::io::Decodable, N: Node<T>>(
                     }
                 }
             }
-            Ok(()) => {
-                trace!("{:?}: wait over", in_channel);
-            }
+            Ok(status) => match status {
+                ChannelReadStatus::ReadReady => trace!("{:?}: wait over", in_channel),
+                _ => {
```

Comment:
If the status is `InvalidChannel`, `Orphaned`, or `PermissionDenied` we reach this arm. I suppose in all these cases we want to return immediately (the status will never be `NotReady` IIUC).

Reply:
Could you enumerate them explicitly in this arm? So if we ever add a new status, we will be forced to reconsider this match statement. Also, based on your comment, it sounds like we want to log an `error` in the `NotReady` case, since I guess it would mean there is a logic error somewhere, right?